optuna_dashboard/ts/components/AppDrawer.tsx

```diff
@@ -120,6 +126,9 @@ export const AppDrawer: FC<{
   const action = actionCreator()
   const [open, setOpen] = useRecoilState<boolean>(drawerOpenState)
   const reloadInterval = useRecoilValue<number>(reloadIntervalState)
+  const studyDetail =
+    studyId !== undefined ? useStudyDetailValue(studyId) : null
```

Comment:
nit: I think you can check both `studyDetail` and `studySummary` to get `is_preferential` property.